packages/common/src/hooks/useDrawer.ts

```diff
@@ -7,15 +7,11 @@ type DrawerController = {
   toggle: () => void;
 };
 
-const localStorageKey = '@openmsupply-client/appdrawer/open';
-
-export const useDrawer = create<DrawerController>(set => ({
-  isOpen: !!JSON.parse(localStorage.getItem(localStorageKey) ?? 'false'),
-  open: () => set(state => ({ ...state, isOpen: true })),
-  close: () => set(state => ({ ...state, isOpen: false })),
-  toggle: () => set(state => ({ ...state, isOpen: !state.isOpen })),
-}));
-
-useDrawer.subscribe(({ isOpen }) => {
-  localStorage.setItem(localStorageKey, JSON.stringify(isOpen));
+export const useDrawer = create<DrawerController>(set => {
```

Comment:
Could continue to keep the choice in local storage but I thought that it was more useful to have the drawer show based on screen size

Reply:
Ok - it is now unresponsive :)
and slightly harder to test.

Required `localStorage.removeItem('@openmsupply-client/appdrawer/open');` in console in order to check the responsiveness after opening and closing like I have.

Reply:
Works well and I like it a lot, thanks